src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -38,6 +38,8 @@ public sealed class LegacyPackageReferenceProject : BuildIntegratedNuGetProject
         private string _projectName;
         private string _projectUniqueName;
         private string _projectFullPath;
+        private IEnumerable<PackageReference> _installedPackages;
```

Comment:
I don't think you need this. 
You assign it and return it. 

It's never used.